core/src/verifier.rs

```diff
@@ -146,91 +160,164 @@ where
 
-        let kickoff_outpoints = kickoff_utxos
-            .iter()
-            .map(|utxo| utxo.outpoint)
-            .collect::<Vec<_>>();
+        // let kickoff_outpoints = kickoff_utxos
+        //     .iter()
+        //     .map(|utxo| utxo.outpoint)
+        //     .collect::<Vec<_>>();
 
         let (recovery_taproot_address, evm_address) = self
             .db
             .get_deposit_info(deposit_outpoint)
             .await?
             .ok_or(BridgeError::DepositInfoNotFound)?;
 
-        let move_commit_tx_handler = TransactionBuilder::create_move_commit_tx(
+        // let move_commit_tx_handler = TransactionBuilder::create_move_commit_tx(
+        //     deposit_outpoint,
+        //     &evm_address,
+        //     &recovery_taproot_address,
+        //     200, // TODO: Fix this
+        //     &self.nofn_xonly_pk,
+        //     &kickoff_outpoints,
+        //     201, // TODO: Fix this
+        //     self.config.network,
+        // );
+
+        // let move_reveal_tx_handler = TransactionBuilder::create_move_reveal_tx(
+        //     OutPoint {
+        //         txid: move_commit_tx_handler.tx.compute_txid(),
+        //         vout: 0,
+        //     },
+        //     &evm_address,
+        //     &recovery_taproot_address,
+        //     &self.nofn_xonly_pk,
+        //     &kickoff_outpoints,
+        //     201, // TODO: Fix this
+        //     self.config.network,
+        // );
+
+        let move_tx_handler = TransactionBuilder::create_move_tx(
             deposit_outpoint,
             &evm_address,
             &recovery_taproot_address,
             200, // TODO: Fix this
             &self.nofn_xonly_pk,
-            &kickoff_outpoints,
-            201, // TODO: Fix this
-            self.config.network,
-        );
-
-        let move_reveal_tx_handler = TransactionBuilder::create_move_reveal_tx(
-            OutPoint {
-                txid: move_commit_tx_handler.tx.compute_txid(),
-                vout: 0,
-            },
-            &evm_address,
-            &recovery_taproot_address,
-            &self.nofn_xonly_pk,
-            &kickoff_outpoints,
-            201, // TODO: Fix this
             self.config.network,
         );
 
         let bridge_fund_outpoint = OutPoint {
-            txid: move_reveal_tx_handler.tx.compute_txid(),
+            txid: move_tx_handler.tx.compute_txid(),
             vout: 0,
         };
-        Ok((
-            kickoff_utxos,
-            move_commit_tx_handler,
-            move_reveal_tx_handler,
-            bridge_fund_outpoint,
-        ))
+        Ok((kickoff_utxos, move_tx_handler, bridge_fund_outpoint))
     }
 
     /// verify burn txs are signed by verifiers
     /// sign operator_takes_txs
-    async fn burn_txs_signed_rpc(
+    async fn burn_txs_signed(
         &self,
         deposit_outpoint: OutPoint,
         _burn_sigs: Vec<schnorr::Signature>,
+        _slash_or_take_sigs: Vec<schnorr::Signature>,
```

Comment:
We should also verify these slash_or_take_sigs

Reply:
Done